README.md

```diff
@@ -62,16 +62,28 @@ that Shortcut URLs will be different across each workspace, as well as between
 the Workspace that you'd like to create the trigger in. Each Workspace has a
 development version (denoted by `(dev)`), as well as a deployed version.
 
-To create a link trigger for the workflow in this template, run the following
-command:
+To create the sample link trigger for the sample workflow in this template, run
+the following command:
 
 ```zsh
 $ slack trigger create --trigger-def triggers/sample_trigger.ts
 ```
 
-After selecting a Workspace, the output provided will include the link trigger
-Shortcut URL. Copy and paste this URL into a channel as a message, or add it as
-a bookmark in a channel of the Workspace you selected.
+Running this command will bring up the workspace selector prompt.
+
+When you select your workspace, you will be prompted to choose an app
+environment for the trigger. The Local app environment means that this trigger
+will only be available when you run the app using the local development server
+that comes with the CLI. The Deployed app environment is what you would select
+if you were readying your Run On Slack app for production deployment.
+
+So that we can interact with our app while running the local development server,
+choose the Local app environment option, and the CLI will finish installing your
+trigger.
+
+Once the CLI is done, the output provided will include the link trigger Shortcut
+URL. Copy and paste this URL into a channel as a message, or add it as a
+bookmark in a channel of the Workspace you selected.
```

Comment:
So, the preceding paragraph covers a lot of what is being suggested as an add here. I'd like to rework that section above, rather than add this new section here, as I think there are some good improvements here! Unfortunately, I cannot directly add suggestions since that code wasn't touched, so it'll have to be done differently. Do you mind if I push changes to your branch?

Reply:
Absolutely do that! These are shared products not the basement scribblings of a mad man. (Okay maybe they're a bit of that too).

Reply:
Entirely possible all of this is unnecessary and we could get away with just a few quick updates, too.